governator-jersey/src/main/java/com/netflix/governator/guice/jetty/GovernatorServletContainer.java

```diff
@@ -1,94 +1,23 @@
 package com.netflix.governator.guice.jetty;
 
-import java.util.Map;
+import com.google.inject.Injector;
+import com.netflix.governator.guice.jersey.GovernatorJerseySupportModule;
+import com.sun.jersey.api.core.ResourceConfig;
 
 import javax.inject.Inject;
+import javax.inject.Named;
 import javax.inject.Singleton;
-import javax.servlet.ServletException;
-
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-
-import com.google.inject.Injector;
-import com.google.inject.Scopes;
-import com.sun.jersey.api.core.ResourceConfig;
-import com.sun.jersey.guice.JerseyServletModule;
-import com.sun.jersey.guice.spi.container.servlet.GuiceContainer;
-import com.sun.jersey.spi.container.servlet.WebConfig;
 
 /**
- * Enhancement to {@link GuiceContainer} by allowing Jersey to be configured via bindings instead of 
- * statically in a {@link JerseyServletModule}.  In addition all singleton resources will be 
- * instantiated eagerly.
- * 
- * <pre>
- * {@code
- * bind(GuiceContainer.class).to(GovernatorServletContainer.class).asEagerSingleton();
- * }
- * </pre>
- * 
- * To set up a Jersey endpoint,
- *
- * <pre>
- * {@code
- * serve("/*").with(GuiceContainer.class);
- * }
- * </pre>
- * 
- * To customize the configuration of the Jersey endpoint
- *
- * <pre>
- * {@code
- * @Provides
- * ResourceConfig getResourceConfig() {
- *     return new PackagesResourceConfig(ImmutableMap.<String, Object>builder()
- *             .put(PackagesResourceConfig.PROPERTY_PACKAGES, "org.example.resources")
- *             .put(ResourceConfig.FEATURE_DISABLE_WADL, "false")   // Enable WADL
- *             .build());
- * }
- * }
- * </pre>
+ * @see GovernatorJerseySupportModule
  */
 @Singleton
-public final class GovernatorServletContainer extends GuiceContainer {
+@Deprecated
+public final class GovernatorServletContainer extends com.netflix.governator.guice.jersey.GovernatorServletContainer {
```

Comment:
Is there any dependency on this class today that cannot break? This is a breaking change.